runtime/vm/VMAccess.cpp

```diff
@@ -1256,6 +1257,9 @@ releaseSafePointVMAccess(J9VMThread * vmThread)
 
 		omrthread_monitor_enter(vm->exclusiveAccessMutex);
 		vm->safePointState = J9_XACCESS_NONE;
+		/* Make sure stale thread pointers don't exist in the stats */
+		vm->omrVM->exclusiveVMAccessStats.requester = NULL;
+		vm->omrVM->exclusiveVMAccessStats.lastResponder = NULL;
```

Comment:
do you need to set `vm->omrVM->exclusiveVMAccessStats.endTime` as well

Reply:
This is a copy of what normal exclusive does, strictly to avoid the crash.  #18318 is open for the full solution.